### DISCUSSION TITLE
File upload restrictions

### DISCUSSION THREAD

**Discussion by JeromeCHA** [GetStream/stream-chat-android#3664] [Q&A] [answered]
Hi

I know there is an option to block/allow files extension and MIME Types in the dashboard, but I would like to know if there is an option to disable the option `recent files` in the attachment screen and forced users to choose only between images and videos?
![image](https://user-images.githubusercontent.com/11457628/173001739-7f107178-8f44-42de-b33c-a6176b9ceaf9.png)

**Comment by JcMinarro** [comment, marked as answer]
Hello @JeromeCHA 
Unfortunately, we don't support it yet on the UI-Components SDK.
Using the Compose SDK you should be able to build your own "attachment picker" without this section, but there is no easy way to remove it at this point.
We will try to add some customizable options over those components in the future.

Thanks for your feedback
